docs/faq_build.md

```diff
@@ -146,7 +123,6 @@ For now, you need to rollback `avr-gcc` to 8 in Homebrew.
 ```
 brew uninstall --force avr-gcc
 brew install avr-gcc@8
-brew link --force avr-gcc@8
```

Comment:
`avr-gcc@8` is keg-only, so it must be manually linked.

Reply:
Resolving this in favor of the other discussion.